src/GovUk.Education.ExploreEducationStatistics.Publisher/Startup.cs

```diff
@@ -126,6 +126,17 @@ public override void Configure(IFunctionsHostBuilder builder)
                         logger: provider.GetRequiredService<ILogger<QueueService>>()))
                 .AddScoped<IReleasePublishingStatusService, ReleasePublishingStatusService>()
                 .AddScoped<IValidationService, ValidationService>()
+                .AddScoped<IPermalinkMigrationService, PermalinkMigrationService>(provider =>
+                {
+                    var connectionString = GetConfigurationValue(provider, "PublicStorage");
```

Comment:
Pull out `GetConfigurationValue(provider, "PublisherStorage")` into a variable here too for consistency, or inline this?

Reply:
Cool - I'll look out for that merge conflict :)